modules/110-istio/images/operator-v1x19x7/patches/apply_go.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/operator/pkg/helmreconciler/apply.go b/operator/pkg/helmreconciler/apply.go
+index 1073df5f14..e1dcb0eb7e 100644
+--- a/operator/pkg/helmreconciler/apply.go
++++ b/operator/pkg/helmreconciler/apply.go
+@@ -235,7 +235,12 @@ func (h *HelmReconciler) serverSideApply(obj *unstructured.Unstructured) error {
+ 	scope.Infof("using server side apply to update obj: %v", objectStr)
+ 	opts := []client.PatchOption{client.ForceOwnership, client.FieldOwner(fieldOwnerOperator)}
+ 	if err := h.client.Patch(context.TODO(), obj, client.Apply, opts...); err != nil {
+-		return fmt.Errorf("failed to update resource with server-side apply for obj %v: %v", objectStr, err)
++		if !(strings.Contains(err.Error(), `cannot patch resource "rolebindings"`) ||
++			strings.Contains(err.Error(), `cannot patch resource "roles"`) ||
++			strings.Contains(err.Error(), `cannot patch resource "clusterroles"`) ||
++			strings.Contains(err.Error(), `cannot patch resource "clusterrolebindings"`)) {
```

Comment:
```suggestion
+		if errors.IsForbidden(err) {
```

Reply:
We have same error type for all these resource: "is forbidden..."

Reply:
`!errors.IsForbidden(...)`

Reply:
Fixed